frontend/src/components/Dashboard.vue

```diff
@@ -0,0 +1,321 @@
+            }
+        },
+    });
+}
+</script>
+
+<template>
+    <ConfirmDialog group="templating">
+        <template #message>
+            <p>
+                Are you sure you want to remove
+                <strong>{{ confirmRemoveMessage.role }}</strong> access to
+                <strong>{{ confirmRemoveMessage.userName }}</strong> in
+                <strong>{{ selectedApplicationDisplayText }}</strong>
+            </p>
+        </template>
+    </ConfirmDialog>
+
+    <div class="row">
+        <div class="col-6">
+            <PageTitle title="Dashboard" subtitle="Manage permissions" />
+        </div>
+        <div class="col-6">
+            <Button
+                v-if="isApplicationSelected"
+                class="dashboard-button"
+                label="Grant new access"
+                @click="router.push('/grant')"
+                ><Icon icon="AddIcon" medium class="icon-size-1"
+            /></Button>
+        </div>
+    </div>
+    <div class="page-body">
+        <div class="row">
+            <div class="col-12">
+                <form
+                    id="selectApplicationForm"
+                    class="form-container dashboard-form"
+                >
+                    <div class="form-group">
+                        <div class="row">
+                            <label
+                                >Select an application you would like to grant
+                                access</label
+                            >
+                            <Dropdown
+                                v-model="selectedApplication"
+                                @change="selectApplication"
+                                :options="filteredOptions"
+                                optionLabel="application_description"
+                                placeholder="Choose an option"
+                                class="application-dropdown"
+                            />
+                        </div>
+                    </div>
+                </form>
+            </div>
+            <div class="col-12">
+                <NotificationMessage
+                    v-if="useNotificationMessage.isNotificationVisible"
+                    severity="success"
+                    :msgText="useNotificationMessage.notificationMsg"
+                    icon="CheckIcon"
+                />
+            </div>
+        </div>
+    </div>
+    <div class="row h-auto" v-if="isApplicationSelected">
+        <div class="col-12">
+            <div class="p-access-table">
+                <div class="p-table-header">
+                    <h3>{{ selectedApplicationDisplayText }} users</h3>
+                    <span>
+                        This table shows all the users in
+                        {{ selectedApplicationDisplayText }} and their
+                        permissions levels
+                    </span>
+                </div>
+
+                <span class="p-input-icon-right">
```

Comment:
this search input border is overlayed by the data table header, add the z-index to this class so it knows which one to show top. So I believe the style for this class is at line 289

.p-input-icon-right {
    ....
    z-index: 1;
    ....
}

![image](https://github.com/bcgov/nr-forests-access-management/assets/23131735/6adb1295-6539-48ff-bafd-5f1981d3f164)